controllers/utils.go

```diff
@@ -158,13 +158,13 @@ func getEnv(key, fallback string) string {
 	return fallback
 }
 
-func CheckIndex(ctx context.Context, indexName string) (bool, error) {
+func (r *FlowTestReconciler) checkIndex(ctx context.Context, indexName string) (bool, error) {
 	logger := log.FromContext(ctx)
 
 	client := &http.Client{}
 
 	// NOTE: When developing this requires port-forward because controller is running locally
-	req, err := http.NewRequest("GET", getEnv("LOG_OUTPUT_ENDPOINT", "http://logging-plumber-log-aggregator.default.svc/"), nil)
+	req, err := http.NewRequest("GET", getEnv("LOG_OUTPUT_ENDPOINT", fmt.Sprintf("http://logging-plumber-log-aggregator.%s.svc/", r.Namespace)), nil)
```

Comment:
why is this an env var?

Reply:
this was only used when we run the operator locally for development work, because this endpoint will be localhost with port-forwarding